src/main/java/nl/ordina/jobcrawler/repo/SkillRepository.java

```diff
@@ -1,23 +1,16 @@
 package nl.ordina.jobcrawler.repo;
 
 import nl.ordina.jobcrawler.model.Skill;
-import nl.ordina.jobcrawler.model.Vacancy;
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.data.jpa.repository.Modifying;
 import org.springframework.data.jpa.repository.Query;
-import org.springframework.data.rest.core.annotation.RepositoryRestResource;
 import org.springframework.stereotype.Repository;
 import org.springframework.transaction.annotation.Transactional;
-import org.springframework.web.bind.annotation.CrossOrigin;
 
 import java.util.List;
-import java.util.Optional;
-import java.util.Set;
 import java.util.UUID;
 
 @Repository
-@RepositoryRestResource
-@CrossOrigin
```

Comment:
what happened to the crossorigin, do we not do that at the moment?

Reply:
it was moved to our own implementation class, the SkillController class